desc/compute/_profiles.py

```diff
@@ -464,10 +500,14 @@ def _p_r(params, transforms, profiles, data, **kwargs):
     transforms={},
     profiles=[],
     coordinates="rtz",
-    data=["p_r", "e^rho"],
+    data=["p_r", "p_t", "p_z", "e^rho", "e^theta", "e^zeta"],
 )
 def _gradp(params, transforms, profiles, data, **kwargs):
-    data["grad(p)"] = (data["p_r"] * data["e^rho"].T).T
+    data["grad(p)"] = (
+        data["p_r"] * data["e^rho"].T
+        + jnp.where(data["p_t"] == 0, 0, data["p_t"] * data["e^theta"].T)
+        + jnp.where(data["p_z"] == 0, 0, data["p_z"] * data["e^zeta"].T)
+    ).T
```

Comment:
e^zeta is finite at axis, so you should be able to leave that alone

Reply:
yeah ill change it. @ddudt `e^theta` however isn't so to get a physical answer we need to make sure we handle that case separately otherwise it returns NaN